_pages/algorithm-development/languages/python.md

```diff
@@ -188,21 +190,7 @@ def apply(input):
       raise AlgorithmError("Please provide a valid input")
 {% endhighlight %}
 
-After you paste the above code into the Algorithmia code editor you can compile and then test the example by passing in a file that you've hosted in [Data Collections](/data/hosted).
-
-Following the example below replace the path to your data collection with your user name (it will appear already if you are logged in), data collection name, and data file name which you can find under "My Collections" in [Data Collections](/data/hosted):
-
-{% highlight python %}
-{"user_file": "data://YOUR_USERNAME/data_collection_dir/data_file.txt"}
-{% endhighlight %}
-
-The code above with return both the original text and the list of each sentence split up into words.
-
-This guide uses a chapter from the public domain book [Burning Daylight, by Jack London](https://en.wikisource.org/wiki/Burning_Daylight), but for brevity we'll only show the first sentence in "text" and "words":
-
-{% highlight python %}
-{"text": "It was a quiet night in the Shovel.", "words": [['It', 'was', 'a', 'quiet', 'night', 'in', 'the', 'Shovel']]}
-{% endhighlight %}
+If you use this code in your own algorithm, you can test it by passing in a file that you've hosted in [Data Collections](/data/hosted). The code above with return both the original text and the list of each sentence split up into words.
```

Comment:
"you've hosted in [Data Collections]" --> "you've uploaded to a [Data Collection]"

Reply:
"the code above with return" --> "the code above *will* return"